doc-tool/src/dotty/tools/dottydoc/model/JavaConverters.scala

```diff
@@ -298,16 +143,88 @@ object JavaConverters {
     def asJava(extras: Map[String, _] = Map.empty): JMap[String, _] = parseEntity(ent, extras)
   }
 
-  private def parseEntity(ent: Entity, extras: Map[String, _]): JMap[String, _] = ent match {
-    case ent: Package   => ent.asJava(extras)
-    case ent: CaseClass => ent.asJava(extras)
-    case ent: Class     => ent.asJava(extras)
-    case ent: Trait     => ent.asJava(extras)
-    case ent: Object    => ent.asJava(extras)
-    case ent: Def       => ent.asJava
-    case ent: Val       => ent.asJava
-    case ent: TypeAlias => ent.asJava
-    case _              => Map.empty.asJava
+  private def parseEntity(ent: Entity, extras: Map[String, _]): JMap[String, _] = {
```

Comment:
Nope, looks like none of the current callers actually pass any extras. I'll remove it!